include/swift/AST/DiagnosticsParse.def

```diff
@@ -702,7 +702,7 @@ ERROR(layout_alignment_should_be_positive,none,
 ERROR(expected_rparen_layout_constraint,none,
       "expected ')' to complete layout constraint", ())
 ERROR(layout_constraints_only_inside_specialize_attr,none,
-      "layout constraints are only allowed inside @_specialize attributes", ())
+      "layout constraints are only allowed inside '_specialize' attributes", ())
```

Comment:
Hmm... I think the other diagnostics (eg the ones for @objc) still have the '@' sign in the message